mesheryctl/internal/cli/root/system/model.go

```diff
@@ -0,0 +1,354 @@
+		if err != nil {
+			cmd.SilenceUsage = true
+			return err
+		}
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return err
+		}
+		ctx, err := mctlCfg.GetCurrentContext()
+		if err != nil {
+			return err
+		}
+		err = ctx.ValidateVersion()
+		if err != nil {
+			return err
+		}
+		return nil
+	},
+	Args: func(_ *cobra.Command, args []string) error {
+		const errMsg = "Usage: mesheryctl system model view [model-name]\nRun 'mesheryctl system model view' to see list of models"
+		if len(args) == 0 {
+			return fmt.Errorf("model name isn't specified\n\n%v", errMsg)
+		} else if len(args) > 1 {
+			return fmt.Errorf("too many arguments\n\n%v", errMsg)
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			log.Fatalln(err, "error processing config")
+		}
+
+		baseUrl := mctlCfg.GetBaseMesheryURL()
+		model := args[0]
+
+		url := fmt.Sprintf("%s/api/meshmodels/models/%s?pagesize=all", baseUrl, model)
+		req, err := utils.NewRequest(http.MethodGet, url, nil)
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+
+		resp, err := utils.MakeRequest(req)
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+
+		// defers the closing of the response body after its use, ensuring that the resources are properly released.
+		defer resp.Body.Close()
+
+		data, err := io.ReadAll(resp.Body)
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+
+		modelsResponse := &models.MeshmodelsAPIResponse{}
+		err = json.Unmarshal(data, modelsResponse)
+		if err != nil {
+			utils.Log.Error(err)
+			return err
+		}
+
+		var selectedModel v1alpha1.Model
+
+		if modelsResponse.Count == 0 {
+			utils.Log.Info("No model(s) found for the given name ", model)
+			return nil
+		} else if modelsResponse.Count == 1 {
+			selectedModel = modelsResponse.Models[0]
+		} else {
+			selectedModel = selectModelPrompt(modelsResponse.Models)
+		}
+
+		var output []byte
+
+		// user may pass flag in lower or upper case but we have to keep it lower
+		// in order to make it consistent while checking output format
+		outFormatFlag = strings.ToLower(outFormatFlag)
+		if outFormatFlag == "yaml" {
+			if output, err = yaml.Marshal(selectedModel); err != nil {
+				return errors.Wrap(err, "failed to format output in YAML")
+			}
+			fmt.Print(string(output))
+		} else if outFormatFlag == "json" {
+			return outputJson(selectedModel)
+		} else {
+			return errors.New("output-format choice invalid, use [json|yaml]")
+		}
+
+		return nil
+	},
+}
+
+// providerCmd represents the `mesheryctl system model` command
+var modelCmd = &cobra.Command{
+	Use:   "model",
+	Short: "Generate and update components",
+	Long:  `Generates and updates components in binary form`,
+	Example: `
+// To view list of components
+mesheryctl system model list
+
+// To view a specific model
+mesheryctl system model view [model-name]
```

Comment:
@zakisk Will you update this to represent actual behaviour?

Reply:
@MUzairS15 sorry, I didn't get you, what do you mean by actual behaviour?

Reply:
Above https://github.com/meshery/meshery/pull/8284/files/f44ef2812ad149efd28314cbb056cf7e10040fbe#diff-934d205e38e560ee0c4bcf41dec8cad7b4fae40a75fc034348bd5b6815953d0cR263 it says generates and update components, but comment says `To view list of components`

Reply:
@zakisk are you going to accept this feedback and incorporate a change?

Reply:
Be sure that you respond to comments with your changes as you resolve comments, so that it is clear what updates were made, @zakisk